specs/am/architecture-management-spec.html

```diff
@@ -59,6 +59,18 @@
             title: "OSLC Architecture Management Version 3.0. Part 2: Vocabulary",
             href: "architecture-management-vocab.html",
           },
+          {
+            title: "OSLC Architecture Management Version 3.0. Part 3: Constraints",
+            href: "architecture-management-shapes.html",
+          },
+          {
+            title: "OSLC Architecture Management Version 3.0. Part 4: Machine Readable Vocabulary Terms",
```

Comment:
Part needs to be removed from here. Could you perhaps add it to the checklist too, please?